lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,20 +15,20 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="../../../assets/etcher.svg"
-            width="150px"
-            height="auto"
+          <svg-icon path="'../assets/etcher.svg'"
+            width="'150px'"
+            height="'auto'"
             os-open-external="https://etcher.io?ref=etcher_offline_banner"></svg-icon>
```

Comment:
Well spotted. From a conceptual point of view, I think it might make sense to have the `os-open-external` on an enclosing `span` or `div`; a bit like the way in HTML you have the `href` attribute on the `A` tag and not on the `IMG` tag.
(but then perhaps react / angular has a different model \*shrug\*)